apps/minifront/src/state/swap/dutch-auction/index.ts

```diff
@@ -12,6 +12,14 @@ import {
 } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/auction/v1/auction_pb';
 import { viewClient } from '../../../clients';
 import { bech32mAssetId } from '@penumbra-zone/bech32m/passet';
+import { sendSimulateTradeRequest } from '../helpers';
+import { fromBaseUnitAmount, multiplyAmountByNumber } from '@penumbra-zone/types/amount';
+import { getDisplayDenomExponent } from '@penumbra-zone/getters/metadata';
+import { Amount } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/num/v1/num_pb';
+import { errorToast } from '@penumbra-zone/ui/lib/toast/presets';
+
+const MAX_OUTPUT_ESTIMATE_MULTIPLIER = 2;
+const MIN_OUTPUT_ESTIMATE_MULTIPLIER = 0.5;
```

Comment:
I know Henry mentioned 2x and 0.5x somewhere, but can't find where now. But that's what these numbers are based on.

Reply:
Seems fine as a scaffold, we can refine later